pages/customized.html

```diff
@@ -1,17 +1,15 @@
 <!-- MAIN CONTENT -->
-<div id="main_content_wrap" class="outer">
-  <section id="main_content" class="inner">
-    <h2><a id="the_Card_Editor" class="anchor" href="#the_Card_Editor" aria-hidden="true"><span class="octicon octicon-link"></span></a>The Card Editor</h2>
-    <p>On the exploration editor page, you will see a graph of interlinked cards. Clicking on any of these cards takes you to the card editor page, which will look something like this: </p>
-    <p><img src="http://wiki.oppia.googlecode.com/git/images/stateEditor.png" width="100%"> </p>
-    <p>This page consists of four parts: </p>
-    <ul>
-      <li><a href="">Parameter changes</a> </li>
-      <li><a href="">Non-interactive content</a> </li>
-      <li><a href="">Interactions</a> </li>
-      <li><a href="">Rules</a> </li>
-    </ul>
-    <p>Each of the above links describes the corresponding part in more detail. </p>
-    <p>The page may also contain statistics about the card. For example, in the screenshot above, we can see a to-do list containing the responses that hit the 'default rule'. If a particular answer appears in this list multiple times, it suggests that writing a separate, more targeted rule for it might be a good idea. You can remove the response from the list at any time by clicking the corresponding 'Delete' text on the right-hand side. </p>
-  </section>
-</div>
+<section id="main_content" class="inner">
+  <h2><a id="the_Card_Editor" class="anchor" href="#the_Card_Editor" aria-hidden="true"><span class="octicon octicon-link"></span></a>The Card Editor</h2>
+  <p>On the exploration editor page, you will see a graph of interlinked cards. Clicking on any of these cards takes you to the card editor page, which will look something like this: </p>
+  <p><img src="http://wiki.oppia.googlecode.com/git/images/stateEditor.png" width="100%"> </p>
```

Comment:
FYI: at some point we should try to migrate the images here too. No urgency though.

Also it would be nice if we could actually update the images -- some of these are pretty out of date!